web-artifacts-io-prjct/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite App</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="side-menu">
+        <h1 class="logo">
+          <a href="#">BRO Client1</a>
+        </h1>
+        <div class="nav">
+          <ul class="nav-items">
+            <a class="nav-list active" href="#">Home</a>
+            <a class="nav-list" href="#">Bank</a>
+            <a class="nav-list" href="#">Settings</a>
+          </ul>
+        </div>
+      </div>
```

Comment:
I suggest you move the sidebar to a separate file and try to connect it using this [method](https://stackoverflow.com/questions/8988855/include-another-html-file-in-a-html-file). That way if you have to change something in the sidebar, you won't have to change it in all the files.